Dockerfile.node

```diff
@@ -0,0 +1,28 @@
+FROM node:12-alpine
+
+COPY rds-combined-ca-bundle.pem redshift-ca-bundle.crt /tmp/
+
+RUN apk update && apk add --upgrade --no-cache coreutils openssl curl && \
```

Comment:
Can this one also be `--virtual`?

Reply:
Nevermind, I misunderstood what `--virtual` does

Reply:
prolly not. we need openssl downstream and curl installs libcurl and wont save us much space... and i would have thought coreutils came as part of the base...but i think we should leave it